src/rgw/rgw_rados.cc

```diff
@@ -7572,7 +7605,8 @@ int RGWRados::copy_obj_data(RGWObjectCtx& obj_ctx,
     }
   }
 
-  ret = processor.complete(etag, mtime, set_mtime, attrs, delete_at);
+  // XXX: need to copy over compression attr and its orig_size here?
+  ret = processor.complete(ofs, etag, mtime, set_mtime, attrs, delete_at);
```

Comment:
(discussed briefly during standup)
i agree that the compression attribute is preserved. i just worry about `ofs` as the first argument to `complete()` - my changes involved passing the uncompressed size here, so the bucket could track both compressed and uncompressed sizes